lib/router.js

```diff
@@ -36,10 +36,13 @@ Cherrytree.prototype.initialize = function (options) {
   this.middleware = []
   this.options = _.extend({
     interceptLinks: true,
-    logError: true
+    logError: true,
+    Promise: global.Promise
```

Comment:
neat, I didn't realise this would work in all envs (browser and node) with the help of webpack